mmseg/models/backbones/cgnet.py

```diff
@@ -0,0 +1,367 @@
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import (ConvModule, build_conv_layer, build_norm_layer,
+                      constant_init, kaiming_init)
+from mmcv.runner import load_checkpoint
+from mmcv.utils.parrots_wrapper import _BatchNorm
+
+from mmseg.utils import get_root_logger
+from ..builder import BACKBONES
+
+
+class FGlo(nn.Module):
```

Comment:
Is this name from paper? If not, we may use a more intuitive one.

Reply:
![image](https://user-images.githubusercontent.com/49829199/97847464-9e80b880-1d2a-11eb-8229-261f0764360f.png)

Yes......